src/foam/crypto/hash/RollingHashingJDAO.java

```diff
@@ -8,14 +8,14 @@
 
 import foam.core.ClassInfo;
 import foam.core.X;
-import foam.dao.AbstractJDAO;
+import foam.dao.JDAO;
 import foam.dao.DAO;
 import foam.dao.MapDAO;
 import foam.lib.json.Outputter;
 import foam.lib.json.OutputterMode;
 
 public class RollingHashingJDAO
-    extends AbstractJDAO
+    extends JDAO
```

Comment:
Why the change here?

Reply:
AbstractJDAO can not load entries to MapDAO now. That is why I changed to extend JDAO